src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -236,6 +255,9 @@ export default class FilterOptionsComponent extends Component {
      * @type {boolean}
      */
     this.allShown = false;
+    if (this.config.storeOnChange) {
+      this.apply();
+    }
```

Comment:
it was to apply the filters on page load, I felt like even though it wasn't specifically asked for in this item since FilterBox was already successfully applying filters on page load, FilterOptions should also do so. I did not add this to the other filters (except for FilterSearch in a past PR because we felt it was necessary for proper backwards compatibility)